opentracing-cdi-example/src/main/java/io/opentracing/contrib/cdi/example/OrderPlacementEvent.java

```diff
@@ -0,0 +1,15 @@
+package io.opentracing.contrib.cdi.example;
+
+import io.opentracing.SpanContext;
+
+public class OrderPlacementEvent {
+    private SpanContext spanContext;
+
+    public OrderPlacementEvent(SpanContext spanContext) {
```

Comment:
Not sure I like the mixing of tracing concern with the business logic here.
Intuitively, the tracing context seems out of place in a business event signature to me.